libp2p/tools/interop/daemon.py

```diff
@@ -56,7 +56,7 @@ def __init__(
         is_pubsub_signing: bool = False,
         is_pubsub_signing_strict: bool = False,
     ) -> None:
-        args = [f"-listen={str(control_maddr)}"]
+        args = [f"-listen={control_maddr!s}"]
```

Comment:
Note: IMO `str(control_maddr)` is more explicit, but never mind.

Reply:
More bytes. 

Also `!s` is proper for brace formatting over `str`, and they're essentially the same thing internally.